pkg/textparse/openmetricsparse_test.go

```diff
@@ -272,71 +272,91 @@ func TestOpenMetricsParseErrors(t *testing.T) {
 		err   string
 	}{
 		{
-			input: "",
+			input: "# EOF",
 			err:   "EOF",
 		},
+		{
+			input: "# EOF\n",
+			err:   "EOF",
```

Comment:
This is also valid input.

Reply:
Same err. We test the error code of the next() and in this case it is EOF.